packages/core/src/styles/layout/_container.scss

```diff
@@ -2,8 +2,13 @@
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
-body[cds-focus-trap-ids*='_'],
-body[cds-layout='no-scrolling'] {
+html[cds-focus-trap-ids*='_'] {
+  overflow: hidden !important;
+  width: 100vw;
+  height: 100vh;
+}
+
+[cds-layout='no-scrolling'] {
```

Comment:
tying these together brought along a lot of baggage for `no-scrolling`. splitting them apart allows `no-scrolling` to be used on containers.

Reply:
Some teams do not use the layout global setup but we still recommend the minimum reset so `cds-focus-trap-ids` might be better in the `module.reset.scss` file.